### PR TITLE
Automatically block users that are running blacklisted workloads

### PR DESCRIPTION
**Pull request by Furisto**
## Description
Users that are running workloads listed in the **very** category will be blocked by agent-smith. This PR also adds support for testing blocking users in preview environments by adding a token and a user for agent-smith (only in preview environments). Before testing had to be done against gitpod.io.

## Related Issue(s)

Fixes [#483](https://github.com/gitpod-io/ops/issues/483)
https://github.com/gitpod-io/ops/pull/1863
https://github.com/gitpod-io/ops/pull/1862

## How to test
- Open workspace
- Run watch date (I have added watch to the list of banned executables in this preview environment)
-> Your workspace should be stopped after a few seconds and your account should have been blocked (in the preview environment, your gitpod.io account should not be affected)

## Notes for reviewers 
- Web App has been added as reviewer because I had to exclude one additional user (the new agent-smith user) from user count, otherwise the first actual user in the preview environment will not have the admin role.

## Release Notes
```release-note
Automatically block users that are running blacklisted workloads
```

### CODE REVIEW

**Review by utam0k — APPROVED** [review]
❤️

File: components/gitpod-db/src/user-db.ts
Comment (geropl) [inline components/gitpod-db/src/user-db.ts:146]: :heart:

Comment (atduarte) [reply components/gitpod-db/src/user-db.ts:146]: Should it be `builtin` instead of `buildin`?

Comment (geropl) [reply components/gitpod-db/src/user-db.ts:146]: nit: the first user is called `buil**t**in`. We should try to avoid future confusion and align the agent-smith user name :+1:

**Comment by geropl** [comment]
Tested and works as advertised. Let's fix [this](https://github.com/gitpod-io/gitpod/pull/9366#discussion_r851965669) before merging, though.

@Furisto Also, we need to configure this for prod before we deploy this (webapp still runs a workspace stack from helm chart!)
Placing a hold here until we have that PR.
/hold

File: components/gitpod-db/src/user-db.ts
Comment (Furisto) [reply components/gitpod-db/src/user-db.ts]: Good catch!

**Comment by Furisto** [comment]
[PR for helm chart update](https://github.com/gitpod-io/ops/pull/1863)
Username has been fixed.
@geropl PTAL

**Review by geropl — APPROVED** [review]

**Comment by geropl** [comment]
/unhold
Will make sure we will merge the helm PR before deployment tomorrow. 👍

**Review by sagor999 — APPROVED** [review]

**Pull request merged**
